05/app.js

```diff
@@ -1 +1,13 @@
-const numbers = [1, 2, 3, 4, 5, 6, 7];
\ No newline at end of file
+const numbers = [1, 2, 3, 4, 5, 6, 7];
+const sumEvenNumbers = numbers.filter(checkEvenNumbers).reduce(sumNumbers, 0);
+console.log(sumEvenNumbers);
+
+function checkEvenNumbers(number) {
+	if (number % 2 === 0) {
+		return number;
+	}
```

Comment:
Dobrze jest napisać jawnie `return false`. Faktycznie domyślnie jest zwracane `undefined` (to też wartość fałszywa) jeśli nic nie napiszmy, ale to dwa różne typu. Lepiej trzymać się tego samego typu w tym pyrzpadku.